src/lib/openjp2/cio.c

```diff
@@ -204,6 +211,7 @@ void OPJ_CALLCONV opj_stream_destroy(opj_stream_t* p_stream)
         if (l_stream->m_free_user_data_fn) {
             l_stream->m_free_user_data_fn(l_stream->m_user_data);
         }
+//else {fprintf(stderr, "ERROR: No m_free_user_data_fn function set\n");}
```

Comment:
Would it be possible to make this a true error? Or is the being kept for future debugging? For future debugging, you could use #ifdef DEBUG...

Reply:
Yes, that looks like an old debugging thing. Probably isn't needed since m_free_user_data_fn() is set by default on creation of l_stream object.